src/main/java/by/dragonsurvivalteam/dragonsurvival/common/handlers/DragonBonusHandler.java

```diff
@@ -137,4 +110,44 @@ public static void addFireProtectionToCaveDragonDrops(BlockDropsEvent dropsEvent
             dropsEvent.getDrops().forEach(drop -> drop.getData(DragonSurvival.ENTITY_HANDLER).isFireImmune = true);
         }
     }
+
+    // We *need* this event to trigger last so that if other mods cancel the event, we don't set the player as jumping
+    @SubscribeEvent(priority = EventPriority.LOWEST)
+    public static void flagPlayersAsJumping(PlayerTickEvent.LivingJumpEvent event) {
```

Comment:
if you want to get cancelled events don't you need to set `(receiveCanceled = true)`?

Reply:
Actually that event cannot be cancelled according to the documentation